sql/core/src/test/scala/org/apache/spark/sql/execution/metric/SQLMetricsSuite.scala

```diff
@@ -182,10 +182,13 @@ class SQLMetricsSuite extends SparkFunSuite with SQLMetricsTestUtils with Shared
   }
 
   test("Sort metrics") {
-    // Assume the execution plan is
-    // WholeStageCodegen(nodeId = 0, Range(nodeId = 2) -> Sort(nodeId = 1))
-    val ds = spark.range(10).sort('id)
-    testSparkPlanMetrics(ds.toDF(), 2, Map.empty)
+    // Assume the execution plan with node id is
+    // Sort(nodeId = 0)
+    //   Exchange(nodeId = 1)
+    //     Range(nodeId = 2)
+    val df = spark.range(9, -1, -1).sort('id).toDF()
```

Comment:
We need to use `range` here? How about just writing `Seq(1, 3, 2, ...).toDF("id")`?

Reply:
Either is fine to me as we now add assert to make sure Sort node exist.

Reply:
Using `Seq` instead of `Range`, we makes things simpler and more readable. I'll change to use `Seq`.